service/src/main/java/com/epam/rft/atsy/service/exception/PasswordValidationException.java

```diff
@@ -1,11 +1,22 @@
 package com.epam.rft.atsy.service.exception;
 
+import java.util.ResourceBundle;
+
 public class PasswordValidationException extends Exception {
+    private String messageKey;
 
     public PasswordValidationException() {
     }
 
-    public PasswordValidationException(String message) {
-        super(message);
+    public PasswordValidationException(String messageKey) {
+        // Both getMessage() and getMessageKey() will return the message key.
+        // getMessageKey() should be used when it helps understanding by its semantics.
+        super(messageKey);
```

Comment:
It could contain a more descriptive message like:

"Password validation failed. Message key is: " +  messageKey

So it will be more readable in the logs. 

Also, as a result of this, the above two comment lines are not needed.